src/subdomains/generic/kyc/services/kyc-log.service.ts

```diff
@@ -55,4 +55,13 @@ export class KycLogService {
 
     await this.kycLogRepo.save(entity);
   }
+
+  async createFileAccessLog(fileName: string, user?: UserData) {
+    const entity = this.kycLogRepo.create({
+      result: `KYC file access: ${fileName}`,
+      userData: user,
+    });
+
+    await this.kycLogRepo.save(entity);
+  }
```

Comment:
Add a new `KycLogType`, create a `KycFileLog` entity which extends `KycLog` and a corresponding repository (see `MergeLog` and `MergeLogRespository`)

Reply:
Add the KYC file ID to the result and the ID of the user who accessed the document (if available, get from JWT)